Library/Formula/weechat.rb

```diff
@@ -28,7 +29,12 @@ class Weechat < Formula
   depends_on "curl" => :optional
 
   def install
-    args = std_cmake_args
```

Comment:
`git rebase -i master` should do the trick for squashing commits. Do "pick" for the first commit and "fixup" for subsequent ones. Or if you have uncommitted changes, just `git commit --amend` will let you include them in the prior commit.